js/server/modules/@arangodb/foxx/manifest.js

```diff
@@ -5,9 +5,15 @@ const joi = require('joi');
 const semver = require('semver');
 const util = require('util');
 const joinPath = require('path').join;
-const arangodb = require('@arangodb');
-const ArangoError = arangodb.ArangoError;
-const errors = arangodb.errors;
+const {
+  plainServerVersion,
+  ArangoError,
+  errors: {
+    ERROR_INVALID_SERVICE_MANIFEST,
+    ERROR_SERVICE_MANIFEST_NOT_FOUND,
+    ERROR_MALFORMED_MANIFEST_FILE
+  }
+} = require('@arangodb');
```

Comment:
This is mostly an aesthetic choice but there's really no reason not to unwrap these variables here directly to avoid redundant destructuring elsewhere.